metrics/network/remote_network/README.md

```diff
@@ -0,0 +1,39 @@
+# Clear Containers remote networking test
+
+- The`remote-networking-iperf3.sh` script measures bandwidth using `iperf3` using a remote setup. Host A will run 
+a container that will act as a server while host B will run a container that will act as a client. 
+The network bandwidth will be measured across the containers.
+
+## Prerequisite
+
+An automatic login from host A (user A) to host B (user B) is needed. You must setup authentication 
+keys to do an ssh login without password.
+
+## Running the remote networking test
+
+`remote-networking-iperf3.sh` should run in host A. The script needs the following inputs to run:
+- Argument of the test to run. In this case this argument is `-b` or `--bandwidth`.
+- Interface name where swarm will run.
+- User of the host B.
+- IP address of the host B.
+
+The `remote-networking-iperf3.sh` test may be run manually:
+
+```
+$ cd metrics
+$ bash metrics/network/remote_network/remote-networking-iperf3.sh "$argument" "$interface_name" "$user" "$ip_address"
```

Comment:
Hi, just a pair of little questions more :), all arguments are mandatory for this script?, then maybe could indicate optional arguments with `[]` or mandatory with `<>` and I think if the user is reading this README file, maybe we could assume the the user is inside `metrics/network/remote_network` directory. This could be an example about how this line could be (removing `cd metrics` line), however  @iphutch could give us a feedback about this:
`bash remote-networking-iperf3.sh "[options]" "<interface name>" "<user>" "<ip addres>" `

and the last point that I would like to touch is about `--bandwidth`, does this option need to be in the first position for working, in general the option can be in whatever argument position however I am not sure if this could cause a conflict with the other arguments, if yes maybe `getopt` could help us with this and maybe the final script could be something like this
`bash remote-networking-iperf3.sh [options] -i <interface name>  -u <user> -a <ip address>`

Reply:
Hi Mario, 
Good points. We can keep a cd line for clarity but cd all the way to the remote_network dir. Typically we use <> when the user is expected to change what is in the example. I suggest the following but leave it to @GabyCT to decide whether to include the "[options]":
```
cd metrics/network/remote_network
bash remote-networking-iperf3.sh "[options]" "<interface name>" "<user>" "<ip address>"
```